test/e2e/apps/replica_set.go

```diff
@@ -102,7 +102,14 @@ var _ = SIGDescribe("ReplicaSet", func() {
 		testReplicaSetServeImageOrFail(f, "private", privateimage.GetE2EImage())
 	})
 
-	ginkgo.It("should surface a failure condition on a common issue like exceeded quota", func() {
+	/*
+		Release : v1.16
+		Testname: Validate ReplicaSet pod limit restrictions
+		Description: Apply quota to the namespace, prohibiting more than two pods.
```

Comment:
Description needs some work, note the difference in:
`create pod with three replicas MUST fail'
vs
`when creating a replicaset of 3, replicasetConditions MUST include the failure X`

Reply:
The description has been updated to be more specific :smile: